package.json

```diff
@@ -55,9 +55,7 @@
 	},
 	"main": "./out/extension",
 	"activationEvents": [
-		"onDebug",
-		"onCommand:extension.bash-debug.getProgramName",
-		"onCommand:extension.bash-debug.selectProgramName"
+		"onDebug"
```

Comment:
I would prefer those actions to be still available. In some cases it is useful + we keep backwards compatibility this way.

So the "initialConfigurations" could be the simple "${file}", but the "configurationSniplets" would include usage of those actions (getProgramName, selectProgramName).